LedgerHQ/ledger-live-desktop
# LL-9044 Change plugin rendering on Manager

## 🦒 Context (issues, jira)
https://ledgerhq.atlassian.net/browse/LL-9044

## 💻  Description / Demo (image or video)
Using the newly exposed `type` for our applications (temporarily leveraging an unused `description` field) we are now able to show a different wording/cta for plugins on the manager. When the changes on the backend API rollout, we will have a proper field to discern which app type we need to render, until then, this is the best we can do.

<img width="1136" alt="image" src="https://user-images.githubusercontent.com/4631227/151536001-cd87bb36-6159-46d4-a3bb-8c07a44e5469.png">

<img width="1312" alt="image" src="https://user-images.githubusercontent.com/4631227/150993484-2a0d959c-d376-4141-9205-0c26a781afff.png">

## 🖤  How to test this
- Enter the manager
- Search for a plugin (ie paraswap)
- Check the rendering matches the screenshots below
- If installed, click on the "Go to App" cta
- It is expected that we should navigate out of the manager and into the discover section

## 🖤  Expectations to reach

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
- **on delivery**: at least one of these two checkboxes must be checked: 

PR must pass CI, merge develop if conflicts, do not force push. Thanks!
